engine/swarm/swarm_manager_locking.md

```diff
@@ -0,0 +1,157 @@
+---
+description: Automatically lock Swarm managers to protect encryption keys
+keywords:
+- swarm, manager, lock, unlock, autolock, encryption
+title: Lock your swarm to protect its encryption key
+---
+
+In Docker 1.13 and higher, the RAFT logs used by swarm managers are encrypted on
```

Comment:
In other docs we seem to use the capitalization `Raft`. This is also what the paper calls it.